packages/flame_bloc/lib/src/flame_bloc_game.dart

```diff
@@ -69,6 +69,9 @@ mixin BlocComponent<B extends BlocBase<S>, S> on Component {
 ///
 /// {@endtemplate}
 class FlameBlocGame extends FlameGame {
+  ///FlameBlocGame contructor with an optional [Camera] as a parameter to FlameGame
+  FlameBlocGame(Camera? camera) : super(camera: camera);
```

Comment:
Can we make this a named parameter to keep consistency with parent class?

Reply:
Done!👍